### PR TITLE
Support adding a separate name for endpoint in healthcheck

### PR DESCRIPTION
**Pull request by guoliang100**
@alainjobart @sougou

### CODE REVIEW

File: go/vt/discovery/celltabletswatcher.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (ctw *CellTabletsWatcher) loadTablets() {
 	ctw.mu.Lock()
 	for key, ep := range newEndPoints {
 		if _, ok := ctw.endPoints[key]; !ok {
-			ctw.hc.AddEndPoint(ctw.cell, ep)
+			ctw.hc.AddEndPoint(ctw.cell, "", ep)
```
Comment (alainjobart) [inline go/vt/discovery/celltabletswatcher.go]: Seems like putting the tablet alias as string would be more appropriate here?

**Comment by alainjobart** [comment]
LGTM. Might want to comment what this name is somewhere though.

File: go/vt/discovery/celltabletswatcher.go (hunk 1 above)
Comment (guoliang100) [inline go/vt/discovery/celltabletswatcher.go]: Added the alias string as the endpoint name.

**Comment by guoliang100** [comment]
Also added the comment for "name"

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**